.github/workflows/e2e-cache-dependency-path.yml

```
@@ -0,0 +1,93 @@
+name: Validate cache with cache-dependency-path option
+
+on:
+  push:
+    branches:
+      - main
+      - releases/*
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  gradle1-save:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Please update checkout version
```suggestion
        uses: actions/checkout@v4
```

---

Would you update the action version along with the other workflows? Thanks.